tests/integration-tests/api.test.ts

```diff
@@ -1010,7 +1010,7 @@ describe('GET /api/address/transactions/[address]', () => {
       }
     }
     const res = await testEndpoint(baseRequestOptions, transactionsEndpoint)
-    expect(res.statusCode).toBe(404)
+    expect(res.statusCode).toBe(200)
```

Comment:
Since it does not return an error anymore, I think is good to also check the integrity of the response; i.e, if it is indeed the address transactions (I think there is another test already doing that, so copying and paste that piece of the code should suffice)

Reply:
I got that error back in place, both in the api and in the test